.evg.yml

```diff
@@ -37,6 +35,131 @@ functions:
     - command: git.get_project
       params:
         directory: realm-cli
+  "setup_mongod":
```

Comment:
did anything change in here from before? I'll glance over it if not.

Reply:
nothing's changed, i copy/pasted all these pieces over from the old `.evg.yml`